### PR TITLE
Make torch version check numeric

### PR DESCRIPTION
**Pull request by jithunnair-amd**
String comparison messes up in cases such as `torch.__version__ = 1.10.blah`

### CODE REVIEW

**Review by datumbox — APPROVED** [review]
LGTM, thanks! Will merge once the CI completes.

File: setup.py [hunk 1]
```diff
@@ -147,7 +147,9 @@ def get_extensions():
     )
 
     is_rocm_pytorch = False
-    if torch.__version__ >= '1.5':
+    TORCH_MAJOR = int(torch.__version__.split('.')[0])
+    TORCH_MINOR = int(torch.__version__.split('.')[1])
+    if TORCH_MAJOR > 1 or (TORCH_MAJOR == 1 and TORCH_MINOR >= 5):
```
Comment (fmassa) [inline setup.py:152]: Should we be [using the newly introduced `TorchVersion` instead](https://github.com/pytorch/pytorch/blob/1022443168b5fad55bbd03d087abf574c9d2e9df/torch/torch_version.py#L13)?

Comment (fmassa) [reply setup.py:152]: Created an issue to track this down https://github.com/pytorch/vision/issues/4296

Comment (ABD-01) [reply setup.py:152]: If we want to use the new [`torch.__version__` as a `TorchVersion` object](https://github.com/pytorch/pytorch/blob/1022443168b5fad55bbd03d087abf574c9d2e9df/torch/torch_version.py#L13) , then just reverting this pr will do the job.
Or can I create a new pr for just one line?

Comment (NicolasHug) [reply setup.py:152]: >  Or can I create a new pr for just one line?

Yup :)

Comment (ABD-01) [reply setup.py:152]: Done.
Thanks Sir

**Pull request merged**